packages/chart/src/components/BarChart.tsx

```diff
@@ -135,10 +176,11 @@ export default function BarChart({ xScale, yScale, seriesScale, xMax, yMax, getX
 
   return (
     <ErrorBoundary component='BarChart'>
-      <Group left={config.runtime.yAxis.size}>
+      <Group left={config.runtime.yAxis.size ? parseInt(config.runtime.yAxis.size) : 66}>
+        {/* left - expects a number not a string */}
         {/* Stacked Vertical */}
         {config.visualizationSubType === 'stacked' && !isHorizontal && (
-          <BarStack data={data} keys={config.runtime.barSeriesKeys || config.runtime.seriesKeys} x={(d: any) => d[config.runtime.xAxis.dataKey]} xScale={xScale} yScale={yScale} color={colorScale}>
+          <BarStack data={cleanData(data)} keys={config.runtime.barSeriesKeys || config.runtime.seriesKeys} x={(d: any) => d[config.runtime.xAxis.dataKey]} xScale={xScale} yScale={yScale} color={colorScale}>
```

Comment:
Can we call cleanData before the component?

Reply:
We could call it on data above all the components if you want
data = cleanedData(data)
but in the file the data comes in as a const so you get an error doing that.
Otherwise would have to map it to another variable and then put that variable as input to the components.
Seemed easier just to call it on the inbound data but I will do whatever you want me to do with it.